### ISSUE TITLE
On Raspbian, I can't start twitter bee

### ISSUE THREAD

**Issue by Riduidel** [muesli/beehive#253]
I'm trying to replace IFTTT by Beehive running on my raspberry 3 (which uses Raspbian Stretch).

Beehive starts normally, but when I try to set up twitter bee, I get this error message

	ERRO[0001] [Riduidel's Twitter]: Error: Could not verify Twitter API Credentials (Get https://api.twitter.com/1.1/account/verify_credentials.json?include_entities=false&skip_status=true&tweet_mode=extended returned status 401, {"errors":[{"code":32,"message":"Could not authenticate you."}]})
	INFO[0001] Fatal bee event: Could not verify Twitter API Credentials 0
	ERRO[0001] [Riduidel's Twitter]: Error: Could not verify Twitter API Credentials (Get https://api.twitter.com/1.1/account/verify_credentials.json?include_entities=false&skip_status=true&tweet_mode=extended returned status 401, {"errors":[{"code":32,"message":"Could not authenticate you."}]})
	INFO[0001] Fatal bee event: Could not verify Twitter API Credentials 1
	ERRO[0001] [Riduidel's Twitter]: Error: Could not verify Twitter API Credentials (Get https://api.twitter.com/1.1/account/verify_credentials.json?include_entities=false&skip_status=true&tweet_mode=extended returned status 401, {"errors":[{"code":32,"message":"Could not authenticate you."}]})
	INFO[0001] Fatal bee event: Could not verify Twitter API Credentials 2
	INFO[0001] Terminating evil bee Riduidel's Twitter after 3 failed tries!

I'm using credentials given by dev.twitter.com application page. Is there something I'm missing/misconfiguring ?

**Comment by Riduidel** [comment]
My raspberry is able to connect to api.twitter.com, so it's not a network error

	curl  https://api.twitter.com/1.1/account/verify_credentials.json?include_entities=false&skip_status=true&tweet_mode=extended
	{"errors":[{"code":215,"message":"Bad Authentication data."}]}

**Comment by Riduidel** [comment]
And notice I've literaly copy/pasted credentials obtained from twitter (and they are visible in the `beehive.conf` file (but I won't show them)

**Comment by Riduidel** [comment]
Interestingly, i've tried using those keys with tweetledee (http://tweetledee.github.io/tweetledee/), installed on the same Raspberry server, and it doesn't work either ! So it seems like the keys are ... not the ones adapted to that kind of application, which is weird.

**Comment by muesli** [comment]
I don't think this is related to Raspbian or the Pi. How/where exactly did you obtain your keys? You essentially need to create a new Application over on https://developer.twitter.com/en/apps

Next you'll need to generate an accesstoken for that newly created application there.

**Comment by Riduidel** [comment]
Yes that's exactly what I did

I have created my application

![image](https://user-images.githubusercontent.com/171018/58536651-7e71cf00-81f1-11e9-88bb-988d5e49b473.png)

And I've created my customer key/secret and access key/secret

![image](https://user-images.githubusercontent.com/171018/58536725-ab25e680-81f1-11e9-87c0-6a9af027ac4a.png)

That I've copied in beehive Twitter bee config

... Well, forget about it, it seems like i've copy/pasted the consumer key with a space after (which was not trimmed by beehive). As a consequence, consumer key was bad.

Do you think it would be possible to always have those keys trimmed in UI ? because sometimes copy/pasting things is ... inefficient

**Comment by Riduidel** [comment]
oh yes it works ! https://twitter.com/riduidel/status/1133642224460283905

**Comment by muesli** [comment]
Yeah, trimming those strings is probably a good idea! I'll rename and reopen the ticket to match that suggestion. Thanks for your feedback!
